Makefile

```diff
@@ -1529,7 +1529,7 @@ else
 	ARCH := $(shell getconf LONG_BIT)
 endif
 
-ifneq ($(wildcard /lib/libc.musl-x86_64.so.1),)
+ifeq ($(shell ldd /bin/sh 2>/dev/null | grep -q musl; echo $$?),0)
```

Comment:
What would be the case if `ldd` was not available? It's not usually installed with musl, you have to manually symlink it. And what if `/bin/sh` does not exist or is a wrapper script?